util/flags.py

```diff
@@ -131,3 +131,6 @@ def create_flags():
                          lambda value: not value or os.path.isdir(value),
                          message='The path pointed to by --export_dir must exist and be a directory.')
 
+
+    # pertubation
+    f.DEFINE_boolean('pertubation', False, 'enable the online audio pertubation')
```

Comment:
This flag is never checked anywhere.

Reply:
Please give this flag a more descriptive name and description. Maybe `apply_time_stretch` for the name and describe it as `randomly time stretch training samples. Requires the Python librosa package to be installed`.